### PR TITLE
Add Dock Tilesize 

### PR DESCRIPTION
**Pull request by moritzsternemann**
This PR adds the `tilesize` key to the Dock section. It can be used to configure the size of the dock icons.

### CODE REVIEW

**Comment by moritzsternemann** [comment]
@yannbertrand it would be awesome if you could run the new record scripts as I don't have the setup with another user ready.

File: record/dock/tilesize/36.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const robot = require('robotjs')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const { compressPngImage } = require('../../utils')
+
+module.exports = {
+  run: async (outputPath) => {
+    console.log('> Recording dock icon size with param set to 36')
+
+    const { stderr: setEnvError } = await exec('defaults write com.apple.screencapture show-thumbnail -bool false && defaults write com.apple.dock tilesize -string bottom && killall Dock')
```
Comment (yannbertrand) [inline record/dock/tilesize/36.js:12]:
```suggestion
    const { stderr: setEnvError } = await exec('defaults write com.apple.screencapture show-thumbnail -bool false && defaults write com.apple.dock tilesize -int 36 && killall Dock')
```

File: record/dock/tilesize/48.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const robot = require('robotjs')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const { compressPngImage } = require('../../utils')
+
+module.exports = {
+  run: async (outputPath) => {
+    console.log('> Recording dock icon size with param set to 48')
+
+    const { stderr: setEnvError } = await exec('defaults write com.apple.screencapture show-thumbnail -bool false && defaults write com.apple.dock tilesize -string bottom && killall Dock')
```
Comment (yannbertrand) [inline record/dock/tilesize/48.js:12]:
```suggestion
    const { stderr: setEnvError } = await exec('defaults write com.apple.screencapture show-thumbnail -bool false && defaults write com.apple.dock tilesize -int 48 && killall Dock')
```

File: defaults.yml [hunk 3]
```diff
@@ -43,6 +43,28 @@ categories:
...
+              height: 463
+            text: Dock icon size of 36 pixels.
+          - value: 48
+            default: true
+            image:
+              filename: "48.png"
+              width: 740
+              height: 463
+            text: Dock icon size of 48 pixels.
+        versions: [Catalina]
```
Comment (yannbertrand) [inline defaults.yml:66]:
```suggestion
        versions: [Big Sur, Catalina]
```

**Review by yannbertrand — CHANGES_REQUESTED** [review]
Looks almost perfect! Thanks for the addition 😄 

Here are the generated screenshots:

**Comment by yannbertrand** [comment]
Quality should be a little better in a zip : [dock-tilesize.zip](https://github.com/yannbertrand/macos-defaults/files/5469204/dock-tilesize.zip)

**Review by yannbertrand — APPROVED** [review]

**Comment by yannbertrand** [comment]
Thank you 🙂

**Pull request merged**